packages/ui/src/Message/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactNode } from "react";
+
+type MessageProperties = {
+  message: string;
+  icon?: string | ReactNode;
+  showIcon?: boolean;
+};
+
+const Message = ({ message, icon, showIcon = false }: MessageProperties) => {
+  const renderIcon = () => {
+    if (showIcon) {
+      return (
+        <span className="icon">
+          {typeof icon === "string" ? <i className={icon} /> : icon}
+        </span>
+      );
+    }
+
+    return null;
+  };
+
+  return (
+    <div className="message-icon-wrapper">
```

Comment:
Since component is named "Message", it'd be nice to have "message" here for className.